Gemfile.lock

```diff
@@ -243,6 +245,7 @@ GEM
     zeitwerk (2.5.4)
 
 PLATFORMS
+  arm64-darwin-21
```

Comment:
The nokogiri automatically stuff was added because you are running a MacBook with arm64 processor. But that's no problem.